src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1351,12 +1351,18 @@ TEST_F(PolicyHandlerTest, OnSnapshotCreated_UrlAdded) {
   EndpointUrls test_data;
   EndpointData data("some_data");
   test_data.push_back(data);
+  AppIdURL next_app_url = std::make_pair("default", "some data");
+  ApplicationSharedPtr mock_app;
 
 #ifdef PROPRIETARY_MODE
   ExtendedPolicyExpectations();
 #else
   EXPECT_CALL(*mock_policy_manager_, GetUpdateUrls("0x07", _))
       .WillRepeatedly(SetArgReferee<1>(test_data));
+  EXPECT_CALL(*mock_policy_manager_, GetNextUpdateUrl(_))
+      .WillOnce(Return(next_app_url));
+  EXPECT_CALL(app_manager_, application_by_policy_id(_))
+      .WillOnce(Return(mock_app));
```

Comment:
@tkireva I will propose you to change to `GetNextUpdateUrl(test_data)` and to `application_by_policy_id("default")`. Or better to `application_by_policy_id(policy::kDefaultID)`.